docs/index.rst

```diff
@@ -26,16 +26,7 @@ Our documentation is structured as follows:
   * :ref:`install-linux-quick`
   * :ref:`install-linux`
 
-Contributing to this documentation
-=======================================================
 
-We welcome collaboration! If you'd like to contribute to our documentation, you can find instructions
-in our `Contributing to ROCm  <https://rocm.docs.amd.com/en/latest/contribute/index.md>`_ section, which discusses:
+To contribute to the documentation refer to `Contributing to ROCm  <https://rocm.docs.amd.com/en/latest/contribute/index.md>`_.
 
-* `Document structure <https://rocm.docs.amd.com/en/develop/contribute/index.html#supported-formats>`_
-* `Toolchains <https://rocm.docs.amd.com/en/develop/contribute/toolchain.html>`_
-* `Documentation builds <https://rocm.docs.amd.com/en/develop/contribute/building.html>`_
-* `How to provide feedback <https://rocm.docs.amd.com/en/develop/contribute/feedback.html>`_
-
-Licensing information for all ROCm components is listed on our
-`Licensing <https://rocm.docs.amd.com/en/latest/about/license.md>`_ page.
+Licensing information can be found at `Licensing <https://rocm.docs.amd.com/en/latest/about/license.md>`_.
```

Comment:
This is passive voice...maybe something like "For licensing information, see..."